### DISCUSSION TITLE
Please add a save option to allow bookmarking an issue or comment

### DISCUSSION THREAD

**Discussion by djnotes** [community/community#12369] [Projects and Issues]
Some issues and comments are very important to me. I would love to have the option to bookmark some issues or some comments for later reference, and have a place like bookmarks in the panel to refer to later on. 
What do you think?

**Comment by rewanthtammana** [comment]
This would be really helpful. Can you please look into this? @github-staff

**Reply by VladimirFokow** [reply]
+1
Why doesn't this already exist? 
It seems like a core functionality - some place for developers to save all the issues which they'd like to work on long-term or in the future

---

[Here](https://github.com/orgs/community/discussions/31058#discussioncomment-3493521) are some suggestions what to do now:

> as for bookmarking, just the regular Bookmark feature of a web browser

**Reply by codespearhead** [reply]
Related: https://github.com/orgs/community/discussions/16331

**Comment by myonlinematters** [comment]
I agree this would be incredbily helpful to me for keeping track of specific issues in repositories with thousands of issues.  What is the stauts of this update?  Has it been implemented?  If so, I don't see it.

**Comment by GoaMind** [comment]
Cross reference to same issue: https://github.com/orgs/community/discussions/16331

**Comment by qwadrox** [comment]
For this specific reason, I created a Chrome extension: https://chromewebstore.google.com/detail/github-issue-glance/kallgfejdofmpelngmpklcddomojplod?authuser=0&hl=en

<img width="977" alt="image" src="https://github.com/user-attachments/assets/554208cb-2f2f-49b2-be2c-03f0bac2e233" />

It allows you to bookmark and manage bookmarked issues and that's it.
